runtime/src/lib.rs

```diff
@@ -1038,7 +1038,8 @@ parameter_types! {
     pub const MaxSubAccounts: u32 = 100;
     pub const MaxAdditionalFields: u32 = 100;
     pub const MaxRegistrars: u32 = 20;
-    pub const MaxAdditionalDataLength: u32 = 128;
+    pub const MaxAdditionalDataLengthXorlessTransfer: u32 = 128;
+    pub const MaxAdditionalDataLengthSwapTransferBatch: u32 = 2000;
```

Comment:
2000 looks too big

Reply:
@vovac12 can you clarify? the task mentioned 2000 bytes

Reply:
It's the upper limit and I think it's not so big, because it's not stored in blockchain storage and only needed to provide additional data for clients and indexers